### DISCUSSION TITLE
Touch control iOS, accidental selecting of text, Music Assistant

### DISCUSSION THREAD

**Discussion by NWSS** [music-assistant/support#1065] [Feature requests and ideas]
Hi, I think the GUI would handle better if the selection of text is not possible. If you touch the wrong way, it starts selecting text instead of handling the UI of Music Assistant. This doesn't happen in the Lovelace Dashboard. So maybe this is something that can be disabled. This will make the GUI a lot more stable I think, no accidental selection of text instead of handling sliders or other controls. Printscreen in attachment. 
![Schermafbeelding 2023-01-13 om 08 49 53](https://user-images.githubusercontent.com/121284555/212267295-9e4ee4a0-54ba-40ee-805c-47b8ec7ad7af.png)

**Comment by OzGav** [comment]
Outdated v1 question
